src/pas/plugins/keycloakgroups/interfaces.py

```diff
@@ -20,7 +20,7 @@ class IKeycloakSettings(Interface):
     )
 
     server_url = schema.TextLine(
-        title=_("Keycloak base url"),
+        title=_("Keycloak base URL"),
         description=_(
             "help_server_url",
             default="Base url for the Keycloak instance to be used",
```

Comment:
Can you check the rest of the files for lower case "url" and replace with uppercase "URL"? I just noticed this one outside of the diff and there may be others.

```suggestion
            default="Base URL for the Keycloak instance to be used",
```

Reply:
Done with the commit 3c95cd4